content/source/docs/enterprise/vcs/github-enterprise.html.md

```diff
@@ -50,22 +50,22 @@ The rest of this page explains the GitHub Enterprise versions of these steps.
     Application Description    | Any description of your choice.
     Authorization callback URL | `https://example.com/replace-this-later` (or any placeholder; the correct URI doesn't exist until the next step.)
 
-4. Click the "Register application" button. This will take you to the application page.
+4. Click the "Register application" button, which creates the application and takes you to its page.
 
 5. Take note of two items: the **Client ID** and the **Client Secret.** You'll copy and paste these unique strings in the next step. Leave this page open in a browser tab.
```

Comment:
Wouldn't hurt to upgrade this one the same as the earlier one:

> Leave this page open in a browser tab. In the next step, you will copy and paste the unique **Client ID** and the **Client Secret.**

Reply:
👍🏼

Reply:
I went and caught all of them, since there was a variant of that everywhere.